src/test/java/uk/gov/hmcts/reform/sendletter/tasks/StaleLettersTaskTest.java

```diff
@@ -47,43 +49,32 @@ void should_do_nothing_when_there_are_no_unprinted_letters() {
         task.run();
 
         // then
-        verify(insights, never()).trackStaleLetter(any(Letter.class));
+        verify(insights, never()).trackStaleLetter(any(BasicLetterInfo.class));
     }
 
     @Test
     @SuppressWarnings("VariableDeclarationUsageDistance")
     void should_report_to_insights_when_there_is_an_unprinted_letter() {
         // given
-        Letter letter = staleLetter();
+        BasicLetterInfo letter = staleLetter();
 
         given(staleLetterService.getStaleLetters()).willReturn(asList(letter));
 
         // when
         task.run();
 
         // then
-        ArgumentCaptor<Letter> captor = ArgumentCaptor.forClass(Letter.class);
+        ArgumentCaptor<BasicLetterInfo> captor = ArgumentCaptor.forClass(BasicLetterInfo.class);
         verify(insights).trackStaleLetter(captor.capture());
 
         // and
         assertThat(captor.getAllValues()).hasSize(1);
         assertThat(captor.getValue().getId()).isEqualTo(letter.getId());
     }
 
-    private Letter staleLetter() {
-        Letter letter = new Letter(
-            UUID.randomUUID(),
-            "checksum",
-            "service",
-            null,
-            "type1",
-            null,
-            false,
-            now()
-        );
-
-        letter.setStatus(LetterStatus.Uploaded);
-        letter.setSentToPrintAt(now().minusDays(1));
+    private BasicLetterInfo staleLetter() {
+        BasicLetterInfo letter = mock(BasicLetterInfo.class);
+        when(letter.getId()).thenReturn(UUID.randomUUID());
```

Comment:
mockito complains if I mock other properties as well (only ID is checked in test)